include/service/memory_entity_service.js

```diff
@@ -80,13 +80,13 @@ module.exports = function MemoryEntityServiceModule(pb) {
         if(value == null && this.site !== GLOBAL_SITE && !this.onlyThisSite) {
             value = getGlobalValue(this, key);
         }
-        cb(null, value);
+        process.nextTick(function() { cb(null, value); });
```

Comment:
This was added to give other lines of execution a chance to run.  We found that when using in memory caching the stack traces were getting too deep.  This just frees things up a bit to tackle the next item in the control flow queue